includes/elements/container.php

```diff
@@ -443,6 +443,28 @@ protected function register_container_layout_controls() {
 			]
 		);
 
+		$this->add_responsive_control(
+			'max_height',
+			[
+				'label' => esc_html__( 'Max Height', 'elementor' ),
+				'type' => Controls_Manager::SLIDER,
+				'size_units' => [ 'px', 'vh', 'custom' ],
+				'range' => [
+					'px' => [
+						'min' => 0,
+						'max' => 1440,
+					],
+					'vh' => [
+						'min' => 0,
+						'max' => 100,
+					],
```

Comment:
Can be removed as this is has already defined in `includes\controls\base-units.php`

Reply:
I just copied it from existing `'Min Height'`. The values are there too :smile: 
I'll remove them :+1: